### PR TITLE
Art.facility lat long migrations

### PR DESCRIPTION
**Pull request by edpaget**
## Description of change
Migrations to enable postgis and add columns for geographic queries of facilities API data.

Related to department-of-veterans-affairs/vets-contrib#368

## Testing planned

Need to ensure this successfully migrates existing facilities data.

#### Unique to this PR
- [x] Enables PostGIS
- [x] Creates point column types
- [x] Migrates existing data into new column
- [x] Updates OS X native install help to move people
- [x] Updates postgres docker image to have postgis 

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by edpaget** [comment]
I think this should be all set to get migrations in.

**Comment by edpaget** [comment]
I will can split that one out. :+1:

File: modules/va_facilities/lib/tasks/facilities_tasks.rake [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # frozen_string_literal: true
-# desc "Explaining what the task does"
-# task :facilities do
-#   # Task goes here
-# end
+
+namespace :va_facilities do
+  desc 'Creates postgis geospatial columns from exiting lat/lng'
```
Comment (patrickvinograd) [inline modules/va_facilities/lib/tasks/facilities_tasks.rake]:
```suggestion
  desc 'Creates postgis geospatial columns from existing lat/lng'
```

**Comment by patrickvinograd** [comment]
LGTM, I was hung up on figuring out how any new facilities would get this column populated, but I see it's in #2365

**Pull request merged**